### PR TITLE
Setup local context resolution for "https://w3id.org/vc-revocation-list-2020/v1"

### PR DESCRIPTION
**Pull request by rbrazhnyk**
Bump SDK to 4.2.0, bump common to 1.6.0

### CODE REVIEW

**Review by Volland — DISMISSED** [review]

File: sdk/browser/README.md [hunk 1]
```diff
@@ -103,8 +103,8 @@ Behaves the same as the wallet-core-sdk `confirmSignIn` method, but with the add
 #### Confirm sign in (if using confirmSignIn for both sign up and login scenarios)
 
 ```ts
-const issueSignupCredential = true
-const { isNew, commonNetworkMember: affinityWallet } = await AffinityWallet.confirmSignIn(token, confirmationCode, options, issueSignupCredential)
+const options = {issueSignupCredential: true  }
```
Comment (DenisPopov15) [inline sdk/browser/README.md:106]: @Volland ideally at least apiKey and env should be passed here as well, so better probably mention that its extension by `issueSignupCredential` general options.

File: sdk/browser/README.md [hunk 2]
```diff
@@ -121,8 +121,8 @@ initialized instance of SDK - `affinityWallet`.
 #### Confirm sign up
 
 ```ts
-const issueSignupCredential = true
-const affinityWallet = await AffinityWallet.confirmSignUp(token, confirmationCode, options, issueSignupCredential)
+const options = { issueSignupCredential: true}
```
Comment (DenisPopov15) [inline sdk/browser/README.md:124]: @Volland the same here as above

File: sdk/expo/README.md [hunk 3]
```diff
@@ -155,12 +155,11 @@ Behaves the same as the wallet-core-sdk `confirmSignIn` method, but with the add
 #### Confirm sign in (if using confirmSignIn for both sign up and login scenarios)
 
 ```ts
-const issueSignupCredential = true
+const options ={ issueSignupCredential: true }
```
Comment (DenisPopov15) [inline sdk/expo/README.md:158]: the same

File: sdk/react-native/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/wallet-react-native-sdk",
-  "version": "4.1.0",
+  "version": "4.2.0",
```
Comment (DenisPopov15) [inline sdk/react-native/package.json:3]: @Volland should we update README at react -native as well

File: common-libs/common/src/_baseDocumentLoader/localContexts.ts [hunk 5]
```diff
@@ -236,4 +236,41 @@ export const localContexts: Record<string, Record<string, any>> = {
       proof: { '@id': 'https://w3id.org/security#proof', '@type': '@id', '@container': '@graph' },
     },
   },
+  'https://w3id.org/vc-revocation-list-2020/v1': {
```
Comment (DenisPopov15) [inline common-libs/common/src/_baseDocumentLoader/localContexts.ts:239]: @Volland could you please update the change log - to describe that performance for revocable VC was improved + documentation updates + minor fixes (as confirmSignup with issue key stone VC option)

**Review by DenisPopov15 — APPROVED** [review]

**Review by vresch — APPROVED** [review]

**Review by vresch — APPROVED** [review]

**Review by DenisPopov15 — APPROVED** [review]

**Pull request merged**